keepalived/vrrp/vrrp_parser.c

```diff
@@ -328,7 +329,7 @@ vrrp_prio_handler(vector_t *strvec)
 
 		vrrp->base_priority = VRRP_PRIO_DFL;
 	}
-	vrrp->effective_priority = vrrp->base_priority;
+	vrrp_set_effective_priority(vrrp, vrrp->base_priority);
```

Comment:
In the context of dynamic priorities being explicit about priorities in logs may prove to be very useful. But the alternative may be indeed the earlier suggestion of being explicit about how the priority changes and the next suggestion about how a received priority compares to ours.